### DISCUSSION TITLE
Missing values of repeatable custom fields in Joomla  3.9.27

### DISCUSSION THREAD

**Discussion by bettyorganero** [joomla/joomla-cms#34534] [Q&A] [answered]
Hello,

After updating my Joomla website to 3.9.27. All the values of my repeatable custom fields are missing in admin and consequently, it is not shown in the frontend. Does anyone know where is the problem? The values are already in the database table fields_values.

Thanks

**Comment by richard67** [comment]
Which version did you update from? 3.9.26? Or older?

**Reply by bettyorganero** [reply]
I think was Joomla 3.9.16 (I'm not sure). But, I did a few checks and realized that when updating the only data was missing in the table fields_values it was the last article published. Is very strange. So I have a backup of the DV I will replace the table.

Thanks

**Comment by bettyorganero** [comment, marked as answer]
So I don't understand why when I updated Joomla this deleted the repeatable field values for my last article published. I only recuperate the table from my backup and all is done. So, close the problem.
Thanks
